irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -124,7 +124,7 @@ namespace iroha {
         log_->info("MST batch prepared");
         // TODO: 07/08/2018 @muratovv rework interface of pcs::propagate batch
         // and mst::propagate batch IR-1584
-        this->pcs_->propagate_batch(*batch);
+        this->pcs_->propagate_batch(batch);
```

Comment:
Isn't the TODO above about the task you did in this PR? If that's the case, you should remove it